<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GravureForm</class>
 <widget class="QWidget" name="GravureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Besoin en Gravure</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FICHE D'EXPRESSION DE BESOIN EN GRAVURE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Date :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="gravure_date_dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_fiche">
       <property name="text">
        <string>N° Fiche CD :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="gravure_numFicheCD_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_demandeur">
       <property name="text">
        <string>Demandeur de Gravure :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="gravure_demandeGravure_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_unite">
       <property name="text">
        <string>Unité :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="gravure_unite_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mediaGroup">
     <property name="title">
      <string>Type de média</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="gravure_cdrom_checkBox">
        <property name="text">
         <string>CD – ROM à Graver</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="gravure_dvdrom_checkBox">
        <property name="text">
         <string>DVD – ROM à Graver</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="softwareGroup">
     <property name="title">
      <string>Logiciels à graver</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_num">
        <property name="text">
         <string>N°</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_logiciel">
        <property name="text">
         <string>Logiciel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_nombre">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>1/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gravure_logiciel1_num_lineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="nombre1">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>2/</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="gravure_logiciel2_num_lineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="nombre2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>3/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="gravure_logiciel3_num_lineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="nombre3"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>4/</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="gravure_logiciel4_num_lineEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="nombre4"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>5/</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="gravure_logiciel5_num_lineEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="nombre5"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_total">
        <property name="text">
         <string>Total :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="gravure_total_lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="motifGroup">
     <property name="title">
      <string>Motif :</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="gravure_motif_textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="gravure_submit_pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Soumettre la demande</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
